Modules/Core/ImageFunction/include/itkRayCastInterpolateImageFunction.hxx

```diff
@@ -1003,29 +1003,29 @@ RayCastInterpolateImageFunction<TInputImage, TCoordinate>::RayCastHelper::Reset(
 
   else
   {
-    for (int i = 0; i < 3; ++i)
+    for (double & it : m_RayVoxelStartPosition)
     {
-      m_RayVoxelStartPosition[i] = 0.;
+      it = 0.;
     }
```

Comment:
I'm a fan of range-based for loops, thanks Hans 👍  However, clang-tidy doesn't always produce the right identifier names. In this case, `it` suggests that it's an iterator, but it's not! I would prefer to call it `coordinate`, for example:

    for (double & coordinate : m_RayVoxelStartPosition) ...

Reply:
Done.